draft-ietf-jsonpath-base.md

```diff
@@ -514,22 +511,22 @@ Unicode code points as described
 in {{RFC3629}}; implementation approaches that lead to an equivalent
 result are possible.
 
-A string to be used as a JSONPath query needs to be *well-formed* and
+A string to be used as a JSONPath query needs to be *well formed* and
 *valid*.
-A string is a well-formed JSONPath query if it conforms to the ABNF syntax in this document.
-A well-formed JSONPath query is valid if it also fulfills all semantic
+A string is a well formed JSONPath query if it conforms to the ABNF syntax in this document.
```

Comment:
English grammar requires the hyphen here.

Reply:
Oops! Fixed.